protected/controllers/AdminController.php

```diff
@@ -692,7 +692,7 @@ public function searchContacts()
 
         $contacts = Contact::model()->findAll($criteria);
 
-        if (count($contacts) == 1) {
+        if (@$_GET['q'] !== '' && @$_GET['q'] !== null && count($contacts) == 1) {
```

Comment:
Is there a reason you're suppressing errors here?

Reply:
It's a fairly standard way of testing the existence of $_GET and $_POST parameters. There are some examples further down in the file. That said, isset() could be used instead of the !== null test. but the empty string equivalence check would still need to be there (but after the isset(), and without the warning suppression)